rebar.config

```diff
@@ -1,14 +1,26 @@
-{require_otp_vsn, "R15|R16|17"}.
+{require_otp_vsn, "R15|R16|17|18"}.
 
 {cover_enabled, true}.
+
 {eunit_opts, [verbose]}.
-{erl_opts, [warnings_as_errors, debug_info, nowarn_deprecated_type]}.
+
+{erl_opts, [
+    debug_info,
+    warnings_as_errors,
+    {platform_define, "^[0-9]+", namespaced_types},
+    {platform_define, "^18", deprecated_now}
+]}.
 {deps, [
-        {riak_pb, "2.1.0.2", {git, "git://github.com/basho/riak_pb", {tag, "2.1.0.2"}}}
-       ]}.
-{edoc_opts, [{stylesheet_file, "priv/edoc.css"},
-             {preprocess, true}]}.
+    {riak_pb, "2.1.0.6", {git, "git://github.com/basho/riak_pb", {branch, "features/lrb/otp-18"}}}
```

Comment:
Note: will change this to updated `riak_pb` tag once basho/riak_pb#160 is merged.